src/components/DateInput.jsx

```diff
@@ -89,7 +92,10 @@ export default class DateInput extends React.Component {
   }
 
   render() {
-    const { dateString } = this.state;
+    const {
+      dateString,
+      isTouchDevice: isTouch,
```

Comment:
That identifier name is already used on [line 5](https://github.com/airbnb/react-dates/pull/336/files#diff-f090ce99cecab0a8a3afc65f42d45527R5)